tests/unit_tests/utils/test_names.py

```diff
@@ -38,8 +38,8 @@ def test_get_list_of_telescope_types():
 
 def test_validate_name():
     with_lists = {
-        "South": ["paranal", "south", "cta-south", "ctao-south", "s"],
-        "North": ["lapalma", "north", "cta-north", "ctao-north", "n"],
+        "South": ["south"],
+        "North": ["north"],
```

Comment:
Do we really need such a validation now that only one string name is accepted?

Reply:
The validation still works with a dict, as the general validation routine in names.py (names._validate_name) requires a dict.